packages/app/src/components/accounts/index.tsx

```diff
@@ -65,33 +78,37 @@ export const Accounts = ({ showAddAccount, accountIndex, next }: AccountsProps)
   if (!wallet) return null;
 
   const accounts = wallet.accounts.map(account => ({
-    username: account.username,
-    address: getGaiaAddress(account),
+    ...account,
+    stxAddress: getStxAddress({ account, transactionVersion: TransactionVersion.Mainnet }),
```

Comment:
hard coded network okay?

Reply:
Have it as a TODO! Still don't have the ChainID persisted in the network store.